l4cv/utils/square_images.py

```diff
@@ -0,0 +1,51 @@
+import os
```

Comment:
Script for cropping images to squares, to maintain aspect ratios when resizing images during training.